### ISSUE TITLE
"become a sponsor" page - banner wording

### ISSUE THREAD

**Issue by f-hwang** [ASSETS-Conference/assets2024#91]
in the banner, "computing and accessibility" should be "computers and accessibility"

**Comment by surajgoraya** [comment]
completed as of #96
